pages/components/Toast/toast.css

```diff
@@ -0,0 +1,91 @@
+:root {
+  --muted-text: #777;
+  --background-white: rgba(255, 255, 255, 0.85);
+  --border-color: rgba(255, 255, 255, 0.85);
+  --box-shadow: rgba(0, 0, 0, 0.15);
+  --border-bottom: rgba(0, 0, 0, 0.05);
+}
+
+.toast {
+  width: 350px;
```

Comment:
Use relative units. Such as rem or %.

Reply:
yes, @05him is correct. That's the intention.  we do not need redundant relative sizing